scripts/run_installer.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -o errexit
+set -o nounset
+
+GPU_INSTALLER_ENV_KEY="cos-gpu-installer-env"
+GPU_INSTALLER_ENV_PATH="/etc/gpu-installer-env"
+
+setup() {
+  /usr/share/google/get_metadata_value "attributes/${GPU_INSTALLER_ENV_KEY}" \
```

Comment:
Do this only if it doesn't exist already (similar to above script):
```
  if [ ! -f "${GPU_INSTALLER_ENV_PATH}" ]; then
...
```

Reply:
Done.